app.go

```diff
@@ -839,12 +840,30 @@ func (app *App) HandlersCount() uint32 {
 }
 
 // Shutdown gracefully shuts down the server without interrupting any active connections.
-// Shutdown works by first closing all open listeners and then waiting indefinitely for all connections to return to idle and then shut down.
+// Shutdown works by first closing all open listeners and then waiting indefinitely for all connections to return to idle before shutting down.
```

Comment:
Why did this change? It should stay in sync with upstream.

Reply:
Prior comments is not grammatically correct.

Reply:
This should be fixed upstream then as well.